src/IW/Core/RepoName.hs

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS_GHC -fno-warn-redundant-constraints #-}
+
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
These two extensions and options are not needed

Reply:
Fixed